app/models/event.rb

```diff
@@ -70,6 +70,14 @@ def attendee_count
     self.signups.count + self.external_signups.count
   end
 
+  def deadline_passed?
```

Comment:
Op het moment gebruik je dit nog nergens, als die plannen er maar wel zijn anders is het een zinloze methode :P

Reply:
Waarom haal je hem dan niet weg 😂